File: app/src/main/java/com/hkdevelopment/bankapp/home/model/Statement.java
package com.hkdevelopment.bankapp.home.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Statement {
    @SerializedName("statementList")
    @Expose
    private List<StatementList> statementList = null;
    @SerializedName("error")
    @Expose
    private Error error;

    public List<StatementList> getStatementList() {
        return statementList;
    }

    public void setStatementList(List<StatementList> statementList) {
        this.statementList = statementList;
    }

    public Error getError() {
        return error;
    }

    public void setError(Error error) {
        this.error = error;
    }
}

File: BankApp/app/src/main/java/com/lucianogiardino/bankapp/domain/model/Statement.kt
package com.lucianogiardino.bankapp.domain.model

import com.google.gson.annotations.SerializedName

data class Statement(

    @SerializedName("title")
    val title: String,

    @SerializedName("desc")
    val desc: String,

    @SerializedName("date")
    val date: String,

    @SerializedName("value")
    val value: Double

)